content/en/tracing/setup_overview/open_standards/otlp_ingest_in_the_agent.md

```diff
@@ -0,0 +1,233 @@
+
+   ```
+   OTEL_EXPORTER_OTLP_ENDPOINT=http://<datadog-agent>:4318.
+   ```
+
+4. Both containers must be defined in the same bridge network, which is handled automatically if you use Docker Compose. Otherwise, follow the Docker example in [Tracing Docker Applications][2] to set up a bridge network with the correct ports.
+
+
+[1]: /agent/docker/
+[2]: /agent/docker/apm/#docker-network
+{{% /tab %}}
+{{% tab "Kubernetes (Daemonset)" %}}
+
+1. Follow the [Kubernetes Agent setup][1].
+
+2. Configure the following environment variables in both the trace Agent container and the core Agent container:
+   
+   For gPRC:
+   ```
+   name: DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT # enables gRPC receiver on port 4317
+   value: "0.0.0.0:4317"
+   ```
+
+   For HTTP:
+   ```
+   name: DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT # enables HTTP receiver on port 55681
+   value: "0.0.0.0:55681"
+   ```
+3. Map the container ports 4317 or 55681 to the host port for the trace Agent container:
+
+   For gPRC:
+   ```
+   ports:
+     - containerPort: 4317
+       hostPort: 4317
+       name: traceportgrpc
+       protocol: TCP
+   ```
+
+   For HTTP 
+   ```
+   ports:
+     - containerPort: 55681
+       hostPort: 55681
+       name: traceporthttp
+       protocol: TCP
+   ```
+
+4. In the application deployment file, configure the endpoint that the OpenTelemetry client sends traces to with the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable:
+
+   For gPRC:
+   ```
+   env:
+    - name: <DD_AGENT_HOST>
+      valueFrom:
+        fieldRef:
+          fieldPath: status.hostIP
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: "http://$<DD_AGENT_HOST>:4317" # sends to gRPC receiver on port 4317
+   ```
+
+   For HTTP:
+   ```
+   env:
+    - name: <DD_AGENT_HOST>
+      valueFrom:
+        fieldRef:
+          fieldPath: status.hostIP
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: "http://$<DD_AGENT_HOST>:55681" # sends to HTTP receiver on port 55681
+   ```
+
+
+[1]: /agent/kubernetes/?tab=daemonset
+{{% /tab %}}
+{{% tab "Kubernetes (Helm)" %}}
+
+1. Follow the [Kubernetes Agent setup][1].
+
+2. Set environment variables for the Agent. You can either use `set` commands:
+
+   For gRPC:
+   ```
+   --set 'agents.containers.traceAgent.ports[0].containerPort=4317,agents.containers.traceAgent.ports[0].hostPort=4317,agents.containers.traceAgent.ports[0].name=traceportgrpc,agents.containers.traceAgent.ports[0].protocol=TCP' 
+
+   --set "datadog.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT,datadog.env[0].value=0.0.0.0:4317"
+   ```
+   For HTTP:
+   ```
+   --set 'agents.containers.traceAgent.ports[0].containerPort=55681,agents.containers.traceAgent.ports[0].hostPort=55681,agents.containers.traceAgent.ports[0].name=traceporthttp,agents.containers.traceAgent.ports[0].protocol=TCP'
+
+   --set "datadog.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT,datadog.env[0].value=0.0.0.0:55681"
+
```

Comment:
```suggestion
```

Reply:
@mackjmr here we have only 2 set statements for gRPC and HTTP, whereas in our doc we have 3 each. Where do we need a correction then?

For gRPC:

1--set 'agents.containers.traceAgent.ports[0].containerPort=4317,agents.containers.traceAgent.ports[0].hostPort=4317,agents.containers.traceAgent.ports[0].name=traceportgrpc,agents.containers.traceAgent.ports[0].protocol=TCP'  
2--set "agents.containers.agent.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT,agents.containers.agent.env[0].value=0.0.0.0:4317" 
3--set "agents.containers.traceAgent.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT,agents.containers.traceAgent.env[0].value=0.0.0.0:4317"

and for HTTP:

1--set 'agents.containers.traceAgent.ports[0].containerPort=55681,agents.containers.traceAgent.ports[0].hostPort=55681,agents.containers.traceAgent.ports[0].name=traceporthttp,agents.containers.traceAgent.ports[0].protocol=TCP' 
2--set "agents.containers.agent.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT,agents.containers.agent.env[0].value=0.0.0.0:4317"
3--set "agents.containers.traceAgent.env[0].name=DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT,agents.containers.traceAgent.env[0].value=0.0.0.0:4317"